testplan/common/config/base.py

```diff
@@ -23,23 +23,32 @@ class DefaultValueWrapper(object):
     Utility class for distinguishing if a value is passed as schema default.
     """
 
-    def __init__(self, value):
+    def __init__(self, value, low_precedence=None):
         self.value = value
+        self.low_precedence = low_precedence
 
     def __repr__(self):
-        return '{}(value={})'.format(self.__class__.__name__, repr(self.value))
+        return '{}(value={}{})'.format(
+            self.__class__.__name__, repr(self.value),
+            ', prefer parent\'s default option' \
+                if self.low_precedence else ''
+        )
 
 
-def ConfigOption(key, default=ABSENT):
+def ConfigOption(key, default=ABSENT, low_precedence=None):
```

Comment:
Would it be better for low_precedence to have a default value as True? Then we toggle the behavior with low_precedence = True/False, rather than None/False which is not very conventional?